src/test/run-make-fulldeps/remap-path-prefix/Makefile

```diff
@@ -0,0 +1,6 @@
+-include ../tools.mk
+
+# Checks if remapping works if the remap-from string contains path to the working directory plus more
+all:
+	$(RUSTC) --remap-path-prefix $$PWD/auxiliary=/the/aux --crate-type=lib --emit=metadata auxiliary/lib.rs
+	! grep "$$PWD/auxiliary" $(TMPDIR)/liblib.rmeta || exit 1
```

Comment:
Can you add the line `grep "/the/aux/lib.rs" $(TMPDIR)/liblib.rmeta`, so that we check that remapping actually does occur (as opposed to no paths being emitted to the .rmeta file at all for some reason).